eng/versioning/version_client.txt

```diff
@@ -18,7 +18,7 @@ com.azure:azure-messaging-eventhubs;5.0.0-preview.5;5.0.0-preview.5
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.0.0-preview.3;1.0.0-preview.3
 com.azure:azure-sdk-template;1.0.3;1.0.3
 com.azure:azure-storage-blob;12.0.1;12.0.1
-com.azure:azure-storage-blob-batch;12.0.1;12.0.1
+com.azure:azure-storage-blob-batch;12.0.2;12.0.2
```

Comment:
For the record: normally incrementing the dependency and current version at the same time would be reserved for something that wasn't released yet otherwise we should be declaring an unreleased dependency. With the aforementioned being said, in this particular case there's nothing in client sdk that has com.azure:azure-storage-blob-batch as a dependency so it's OK here.

Reply:
While there isn't anything currently that depends on it if a dependency creeps in later we would still want it to be the earlier version so I'd definitely consider putting that back to the older version.

Reply:
This is for a potential patch release, would it be likely that we back port an entire module which depended on this? I agree in the future that this shouldn't have been the versioning change made.

Reply:
I missed that this is in a hotfix branch. Given that I think we are ok to keep this change but in the future I'd recommend only updating the current version unless there is a reason to update the dependency versions.